native/thrift_message_read_module.c

```diff
@@ -0,0 +1,328 @@
+#include <Python.h>
+#include <string.h>
+#include "thrift_struct_read_module.h"
+#include "thrift_consts.h"
+#include "thrift_utils.h"
+
+char *readMessageBegin(char *,char *,int *,int *);
+void readMessageEnd(char*,char*);
+char *message_type_to_str(int);
```

Comment:
these should be static (since they are only used in this file, thus the symbols should not be public)